tools/devops/automation/templates/release/vs-insertion-prep.yml

```diff
@@ -168,3 +159,38 @@ stages:
             inputs:
               artifactName: ${{ parameters.uploadPrefix }}vsdrop-multitarget-signed
               downloadPath: $(Build.StagingDirectory)\$(MultiTargetVSDropCommitStatusName)
+
+      - script: make -C $(Build.SourcesDirectory)/dotnet version-props
+        displayName: make version props
+
+      - task: DotNetCoreCLI@2
+        displayName: generate and publish BAR manifest
+        inputs:
+          projects: $(Build.SourcesDirectory)/dotnet/package/Microsoft.iOS.Ref/package.csproj
```

Comment:
Does this work if we're not building for iOS?

Pushing a diff like this will test if a single platform works: https://gist.github.com/rolfbjarne/6235268e8fb6bc32ea7c74ab724c5f8a

If the code is truly platform-agnostic, maybe create a new, platform-agnostic csproj instead.

Reply:
I tested this locally and I don't see any issues with iOS disabled, the project file always exists and the ios props file is still generated with `make version-props`. I kicked off this test build for CI as well: https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9660668&view=results

Reply:
Good thing I queued the build 😆, looks like the props files aren't generated for all platforms. Let me try to fix this up.

Reply:
Latest test run is here - https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9661904&view=results, I updated the script to build the project associated with the first platform that is enabled.

Reply:
Looks like this worked and pushed only the macos packs https://devdiv.visualstudio.com/DevDiv/_build/results?buildId=9661904&view=logs&j=45f1b321-fe00-5774-563c-3f6362fe97e5&t=d5aaaad1-912f-5aa2-63da-3a7555914b3b, I'll revert the testing changes